src/app/components/Routes/Routes.tsx

```diff
@@ -13,13 +13,13 @@ export function Routes() {
 
   return (
     <Switch>
-      <Route path={prefixRouteWithPluginBaseUrl(ROUTES.PROFILES_EXPLORER_VIEW)} exact>
```

Comment:
Why the change?

Reply:
`exact` param is not available in react router 6:

```
ERROR in ./src/app/components/Routes/Routes.tsx:22:67
TS2322: Type '{ children: Element; path: string; exact: true; }' is not assignable to type 'IntrinsicAttributes & RouteProps'.
  Property 'exact' does not exist on type '(IntrinsicAttributes & PathRouteProps) | (IntrinsicAttributes & LayoutRouteProps)'.
    20 |         <AdHocView />
    21 |       </Route>
  > 22 |       <Route path={prefixRouteWithPluginBaseUrl(ROUTES.SETTINGS)} exact>
       |                                                                   ^^^^^
    23 |         <SettingsView />
    24 |       </Route>
    25 |       {/* Default Route */}
```

from [docs](https://reactrouter.com/en/main/upgrading/v5#relative-routes-and-links):

> `<Route exact>` is gone. Instead, routes with descendant routes (defined in other components) use a trailing * in their path to indicate they match deeply

Reply:
Then we have to prevent discrepancies between the local environment/devx and the other environments by also upgrading React Router in the npm dependencies.

Reply:
I have reverted the change and added explicit react-router version in package.json. Grafana 11.3.0 provides v5. Locally if we don't specify react-router it starts pulling v6 to node_modules 🤔

Reply:
> it starts pulling v6 to node_modules
Transitive dependencies FTW!